### ISSUE TITLE
feature request: save packages report to a file

### ISSUE THREAD

**Issue by boussaffawalid** [conan-io/conan-package-tools#392]
It would be very helpful if you could add an option to generate/save a report of generated packages details.

Thanks

**Comment by danimtb** [comment]
Maybe store the json output? https://docs.conan.io/en/latest/reference/commands/output/install.html

`conan create . user/channel --json report.json`

Do you think that be valid for your use case?

**Comment by boussaffawalid** [comment]
@danimtb not really, what I meant something similar to what the conan-package-tools print already to console but with the option to save it to file. 

 +---------------+--------------------+--------------------+--------------+--------+
 | compiler      | compiler.runtime   |   compiler.version | build_type   | arch   |
 |---------------+--------------------+--------------------+--------------+--------|
 | Visual Studio | MD                 |                 14 | Release      | x86_64 |
 | Visual Studio | MDd                |                 14 | Debug        | x86_64 |
 +---------------+--------------------+--------------------+--------------+--------+


Context: we use a build.py script to package a lot of projects in CI and we control this build script using the different environment variables that conan-package-tools provide.
However, there is no way currently to figure out what was build exactly.
 
A report should contains for example:
- Full package name
- Different build types
- Different archs
- Remote channel where the packages were uploaded
- build time will be very helpful as well 
.....

**Comment by boussaffawalid** [comment]
@uilianries lockfiles provide somehow these information, however I'm not sure how to integrate that with CPT
